src/main.zig

```diff
@@ -1,7 +1,11 @@
 const std = @import("std");
 const ChildProcess = std.ChildProcess;
 
+// would something like this make sense?
+const CdFlags = enum { L, P };
+
 const Argv = std.ArrayList([]const u8);
+const Flags = std.ArrayList(CdFlags);
```

Comment:
Not all flags would be CdFlags. IMO, you can just replace Flags in the code with std.ArrayList(CdFlags) if you want. If not, please rename this to show that it is cd.

Reply:
flags are gone for now